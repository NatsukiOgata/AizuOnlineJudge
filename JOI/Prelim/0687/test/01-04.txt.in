50
23 559 41 142 300 993 531 140 484 617 746 230 324 807 271 842 996 502 615 768 205 997 572 228 413 306 386 675 774 976 957 672 935 201 812 219 77 274 160 209 571 583 592 345 885 817 764 451 312 10
