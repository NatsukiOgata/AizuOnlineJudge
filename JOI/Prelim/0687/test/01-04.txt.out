10136
14176
